ulog/context.go

```diff
@@ -74,8 +74,7 @@ func (t trace) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 	if !j.IsValid() {
 		return fmt.Errorf("invalid span: %v", j.SpanID())
 	}
-	enc.AddUint64("span", j.SpanID())
-	enc.AddUint64("trace", j.TraceID())
-	enc.AddUint64("parent", j.ParentID())
```

Comment:
Note: I think in most cases parent is not useful, maybe we can have a configurable way to turn it on in future if service owners want to.